app/auto-eligibility/telemetry/raise-telemetry-event.js

```diff
@@ -0,0 +1,23 @@
+const raiseEvent = require('../../event/raise-event')
+const appInsightsConfig = require('../../app-insights/app-insights.config')
+
+module.exports = (customer) => {
+  return async (type, message, data = customer) => {
+    await raiseEvent({
+      name: 'send-session-event',
+      properties: {
+        id: `${customer.sbi}_${customer.crn}`,
+        sbi: customer.sbi,
```

Comment:
Needs to be a string https://github.com/DEFRA/ffc-ahwr-event/blob/6551b6056b78018857f99f01b66dacdf57027a48/ffc-ahwr-event/event-schema.js#L7